go.sum

```diff
@@ -1,312 +1 @@
-github.com/pulumi/pulumi-hugo/themes/default v0.0.0-20210420180605-5a9c775a3d1c h1:ODVskkF6w7jbwIg67q4aOsc9N5524AoyA0QhJnSnK+0=
```

Comment:
So I am about to merge this PR because I don't want to keep having to rebase this PR every time there is a new docs site build and there is a conflict with this `go.sum`. I do have a question about this file's contents in `master`, though. Why does it have every single past version of the `pulumi-hugo` module in it? It should only have the current version, correct? I ran `go mod download` on my machine and that knocked out all the previous versions and only the current version referenced by `go.mod` is in here now. cc @cnunciato

Reply:
I went ahead and reverted this for now since this seems to be something we should fix in our cron workflow that updates this file. I didn't want to introduce a large diff in the next automated PR for this file.